.github/workflows/sequential-test-fedora-binaries copy.yml

```diff
@@ -0,0 +1,147 @@
+name: Test Fedora binaries (Sequential)
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - PruStephan/Add_tests_for_packages
+
+jobs:
+  test_binaries:
+    name: Install and test binaries
+    runs-on: [self-hosted, Linux, X64, nix-with-docker]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up and test tezos-client
+        run: |
+          docker run --rm -i fedora:latest /bin/bash -c "
+            sudo dnf update -y
+            sudo dnf install -y 'dnf-command(copr)'
+            sudo dnf copr enable -y @Serokell/Tezos
+            sudo dnf install tezos-client -y
+            sudo tezos-client --version
+          "
```

Comment:
AFAIR @pasqu4le wanted this to be able to run locally too. 
So I propose you to develop `Dockerfile` and a wrapper script to run all of this conveniently.
Please refer to the `docker/package/Dockerfile-ubuntu`, `docker/package/Dockerfile-fedora` and `docker/build/Dockerfile`. `docker/docker-static-build.sh` could be also interesting for you.

Reply:
@pasqu4le also, do you think it's a good idea to use GA? AFAIU we use it only for bottle builds on `monterey`. Since all other builds run on `buildkite` and we use `docker`, I don't see any benefit from using GA here.

Reply:
> AFAIR @pasqu4le wanted this to be able to run locally too.

Yes, indeed, and we need documentation on how to do so too.

Anyone should be able to follow the instruction and (after meeting the requirements) run these tests locally.

_BTW, that's true in general, about almost everything that runs in CI._

> @pasqu4le also, do you think it's a good idea to use GA? 

AFAIK the OPS team intends to stop using buildkite entirely at some point (cc @rvem, can you confirm?), so it's not a bad idea either.

Unless there's some reason to prefer buildkite now, I think that it's fine to implement new actions in GA (especially if they aren't connected to what we do in buildkite).

Reply:
> AFAIK the OPS team intends to stop using buildkite entirely at some point (cc @rvem, can you confirm?)

Well, not really, public buildkite is free and we're rather considering not using it for new projects and maybe eventually migrating existing projects to it. Though, `tezos-packaging` has very complicated CI setup, so I'm not sure if it's actually worth completely switching it